include/iso15118/d20/state/dc_charge_loop.hpp

```diff
@@ -18,6 +22,10 @@ struct DC_ChargeLoop : public FsmSimpleState {
     float present_current{0};
     bool stop{false};
 
+    std::optional<std::time_t> new_departure_time;
+    std::optional<uint8_t> new_target_soc;
+    std::optional<uint8_t> new_min_soc;
+
```

Comment:
Why not just use `UpdateDynamicModeParameters` here?

Reply:
Done